Cargo.toml

```diff
@@ -10,6 +9,7 @@ readme = "README.md"
 license = "MIT"
 authors = ["Matthew Hall <matthew@quickbeam.me.uk>"]
 edition = "2018"
+include = ["src/**/*.rs", "README.md", "LICENSE"]
```

Comment:
Might it make sense to include the `CHANGELOG.md`?

Reply:
Maybe, not sure, since these are files that are going to be uploaded to the package. Normally an user that'd like to see all the new changes would check out the repo, ig.